main.go

```diff
@@ -84,6 +97,7 @@ func createSFCommentBody(post *sfapi.DiscussionPost, ticket *sfapi.Ticket) (stri
 
 func addCommentToIssue(ctx context.Context, post sfapi.DiscussionPost, ticket *sfapi.Ticket, issue *github.Issue) (*github.IssueComment, error) {
 	body, err := createSFCommentBody(&post, ticket)
+	verbose(fmt.Sprintf("  Adding comment to issue"))
```

Comment:
```suggestion
	verbose("  Adding comment to issue")
```

You don't need to do `Sprintf` if you don't use any formating directive